### PR TITLE
Update entire structure of learn docs

### PR DESCRIPTION
**Pull request by tgberkeley**
The page links that are in the sidebar that are not accordions need to be styled a little better

### CODE REVIEW

File: docs/events/chaining_events.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+# Chaining events
+
+## Calling Event Handlers From Event Handlers
+
+You can call other event handlers from event handlers to keep your code modular. Just use the `self.call_handler` syntax to run another event handler. As always, you can yield within your function to send incremental updates to the frontend.
+
+
+```python exec
+
+class CallHandlerState(State):
```
Comment (Alek99) [inline docs/events/chaining_events.md:18]: Example seems to be broken for me

File: docs/vars/var-operations.md [hunk 2]
```diff
@@ -8,7 +8,6 @@ import numpy as np
 import reflex as rx
 
 from pcweb.base_state import State
-from pcweb.pages.docs.advanced_guide.custom_vars import custom_vars
 from pcweb.templates.docpage import (
```
Comment (Alek99) [inline docs/vars/var-operations.md:11]: Not related to this pr but just noticed the [Supported Operations](http://localhost:3000/docs/vars/var-operations/#supported-operations) example is not working

File: docs/utility_methods/utility_methods_overview.md
Comment (Alek99) [inline docs/utility_methods/utility_methods_overview.md:1]: For this section we can probably break it up into two files router methods and other methods

File: docs/pages/page_meta_data.md
Comment (Alek99) [inline docs/pages/page_meta_data.md:1]: This file can be renamed just meta data

File: pcweb/pcweb.py
Comment (Alek99) [inline pcweb/pcweb.py:1]: Need to uncomment this

**Review by Alek99 — CHANGES_REQUESTED** [review]
Nice a few comments we will also have to add dynamic rendering section once we merge in your other two prs

File: pcweb/pcweb.py
Comment (tgberkeley) [reply pcweb/pcweb.py:1]: yes I just noticed this, will uncomment now

File: docs/events/chaining_events.md (hunk 1 above)
Comment (tgberkeley) [reply docs/events/chaining_events.md:18]: fixed

File: docs/vars/var-operations.md (hunk 2 above)
Comment (tgberkeley) [reply docs/vars/var-operations.md:11]: what is broken with it? It seems to work for me.

File: docs/utility_methods/utility_methods_overview.md
Comment (tgberkeley) [reply docs/utility_methods/utility_methods_overview.md:1]: changed

File: docs/pages/page_meta_data.md
Comment (tgberkeley) [reply docs/pages/page_meta_data.md:1]: changed

File: pcweb/pcweb.py
Comment (tgberkeley) [reply pcweb/pcweb.py:1]: uncommented

**Comment by Alek99** [comment]
Last think think the secitons should be names UI, State. Not frontend and backend

File: docs/components/component_props.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ```python exec
```
Comment (picklelo) [inline docs/components/component_props.md:1]: Can this page be called `props.md` or is that a name conflict?

File: docs/pages/meta_data.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+```python exec
```
Comment (picklelo) [inline docs/pages/meta_data.md:1]: Should be `metadata.py` as one word

File: docs/assets/referencing-assets.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+```python exec
```
[the remark this answers is not in the document]
Comment (tgberkeley) [reply docs/assets/referencing-assets.md:1]: changed

File: docs/components/component_props.md (hunk 3 above)
Comment (tgberkeley) [reply docs/components/component_props.md:1]: renamed

File: docs/pages/meta_data.md (hunk 4 above)
Comment (tgberkeley) [reply docs/pages/meta_data.md]: done

**Review by picklelo — APPROVED** [review]

**Pull request merged**